### ISSUE TITLE
Connections should only be shown for users with admin permissions

### ISSUE THREAD

**Issue by marckhouzam** [pantsel/konga#92]
The page showing the list of connections managed by Konga currently shows the API key used to connect to the Kong admin API.  This is privileged information which should not be show to non-admin users.  Using this information as shown in the Connection page, any user can send admin commands to the Kong server, which is not wanted.

It would be nice to hide the API keys from the Connections page.

However, even the information of the Admin URL of the Connections page could be considered privileged.  I think the entire Connections page should only be available for admin users.

To complete this effort, the connection selector shown at the bottom-right should be modified to only show the name of the connection, and nothing else.

**Comment by pantsel** [comment]
Hi @marckhouzam and thanks for the feedback!

You've made some valid points there. 

I think it all comes down to the way Konga is to be used.

When I designed it, I did it for internal-company use only and didn't feel the need of hiding "privileged" information since only devs where using the app and the information was important to everyone. 
As a matter of fact, the entire ACL system was kinda meaningless as well. :)

That being said, I understand the way you see it. I'll make some modifications with the next release and keep you posted.

**Comment by pantsel** [comment]
Hiding priviledged information can now be fully accomplished with the latest commits in conjunction with changing  user permissions in the settings page.

**Comment by marckhouzam** [comment]
Thanks for the quick turnaround!  I will be trying you the new version shortly.

**Comment by marckhouzam** [comment]
I am unable to test version 0.8.5 or latest using docker on windows.  I get the following error:

>docker run -p 1337:1337 --rm --name konga pantsel/konga:0.8.5
debug: Hook:api_health_checks:process() called
debug: Hook:health_checks:process() called
debug: Hook:user_events_hook:process() called
/app/node_modules/sails-hook-validation/index.js:34
        (_ || sails.util._)
         ^

ReferenceError: _ is not defined
    at patch (/app/node_modules/sails-hook-validation/index.js:34:10)
    at /app/node_modules/sails-hook-validation/index.js:101:21
    at /app/node_modules/sails/lib/app/private/after.js:91:14
    at /app/node_modules/async/lib/async.js:721:13
    at /app/node_modules/async/lib/async.js:52:16
    at done (/app/node_modules/async/lib/async.js:246:17)
    at /app/node_modules/async/lib/async.js:44:16
    at Sails.<anonymous> (/app/node_modules/async/lib/async.js:718:17)
    at Sails.<anonymous> (/app/node_modules/async/lib/async.js:167:37)
    at Sails.g (events.js:292:16)
    at emitNone (events.js:91:20)
    at Sails.emit (events.js:185:7)
    at Sails.emitter.emit (/app/node_modules/sails/lib/app/private/after.js:50:11)
    at /app/node_modules/sails/lib/app/private/loadHooks.js:103:15
    at /app/node_modules/sails-hook-orm/node_modules/async/lib/async.js:52:16
    at /app/node_modules/sails-hook-orm/node_modules/async/lib/async.js:550:17
    at /app/node_modules/sails-hook-orm/node_modules/async/lib/async.js:544:17
    at _arrayEach (/app/node_modules/sails-hook-orm/node_modules/async/lib/async.js:85:13)
    at Immediate.taskComplete (/app/node_modules/sails-hook-orm/node_modules/async/lib/async.js:543:13)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)

**Comment by marckhouzam** [comment]
I was able to use pantsel/konga:latest on Linux, but not version 0.8.5 which fails with the same error as on Windows.

So I was able to test the nice improvements you made.
Things look great. I like that I can either hide the entire Connections page (which I hadn't realized I could do before) or now leave that page but the API keys are now hidden.  And I see that the list of connections at the bottom, now does not show the url.  

I will close the issue as it does exactly what was requested.  Thank you.

However, it was reported to me that the api key is sent in plain text from the browser to the konga server, so someone resourceful could find that api key and get admin privileges.  I will open a new issue about that, as it is surely more complicated.
